packages/mapviewer/src/modules/map/components/cesium/CesiumInteractions.vue

```diff
@@ -233,6 +243,62 @@ function onClick(event) {
     viewer.scene.requestRender()
 }
 
+function create3dKmlFeature(viewer, kmlFeature, kmlLayer) {
+    let geometry
+    let featureCoordinates
+    if (kmlFeature.id.position) {
```

Comment:
are we 100% sure `kmlFeature` and `kmlFeature.id` will be defined? If not we could watch our backs by adding a little `?` between each selectors

Reply:
or have an early empty return
```Javascript
if (!kmlFeature || !kmlFeature.id) {
    return
}
```